### ISSUE TITLE
Wrong version on npm?

### ISSUE THREAD

**Issue by mjackson** [SlexAxton/messageformat.js#90]
This is strange:

```
$ npm install --save messageformat
$ cat node_modules/messageformat/package.json | grep version
  "version": "0.2.1",
$ cat node_modules/messageformat/messageformat.js | grep @version
 * @version 0.1.7
```

Version 0.2.1 of the npm package looks like it actually contains version 0.1.7 of `messageformat.js`.

**Comment by SlexAxton** [comment]
Probably my inability to use build tools correctly. I'll push something out
asap.

**Comment by mjackson** [comment]
Thanks @SlexAxton !

**Comment by SlexAxton** [comment]
As an update to this issue, since it's been way too long:

- The 0.2.1 file contains the 0.2.1 library, but with the wrong version in the header comment. So this isn't as sad as I thought, luckily.
- We've got a 3.0.0-0 tag out now that we're not quite ready to publish without docs, but you can install it via tag.
- The newest versions make this problem much harder to occur in the future.

I'll see how easy it is for me to check out this old tag and push the comment change as 0.2.2... but just wanted to update this thread for completeness sake.

**Comment by SlexAxton** [comment]
Went ahead and released an `0.2.2` for good measure - but it has no functionality changes. Thanks!
